### ISSUE TITLE
Simple CPU Monitor

### ISSUE THREAD

**Issue by acritum** [gnemonix/factorio-prod-calc#1]
Sorry for leaving a message here but I can't find any other way to contact you. I used Simple CPU Monitor for a few days and found a bug. Could you please revise it:

Mint 18.3 cinnamon 64 bit. Bug: after resuming from hibernate, the value freezes and doesn't update. Besides, the length of the applet in the panel is VERY long, about 3-4 lengths of "100%" label (much longer than in the screenshot). I don't know what is it for and how to reduce the length. Please fix. The plugin is really useful, but needs improvements.

You can contact me at spamaway [@] mail [.] ru if you have further questions and you can delete this message from here when you have read it. Thanks.

**Comment by gnemonix** [comment]
@acritum Thank you for contacting me. I'll get right on it. (Presently the applet doesn't even run on my computer.)

For the future, please report cinnamon applet bugs on the correct repository. Actually, please re-post this over there.

See https://github.com/linuxmint/cinnamon-spices-applets/blob/master/.github/CONTRIBUTING.md for proper bug reporting.

And thanks for using my applet, I didn't think it was still in use. :)

--
gnemonix

**Comment by acritum** [comment]
Thank you. I think many people use your applet because it's the only applet that shows numbers. The other applets available in cinnamon show graphs. I installed it because I had strange CPU usage 8-10% when idle and I tried to get rid of it and your applet is very handy to control the effect of disabling various stuff. Now I seem to disabled something and the CPU usage lowered to 0-3% when idle which seems to be OK. Anyway, I think I'll keep your applet in the panel just in case :-) I re-posted the problem here and attached a screenshot:
https://github.com/linuxmint/cinnamon-spices-applets/issues/1744
